### DISCUSSION TITLE
How to clear images produced by icat with unicode placeholder method?

### DISCUSSION THREAD

**Discussion by BigBrotherJu** [kovidgoyal/kitty#6477] [Q&A] [answered]
When icat produces images with normal methods (like `--transfer-mode file`), these images can be cleared by `--clear`. But `--clear` does not clear images produced by `--unicode-placeholder`. Is there any way to clear images produced by `--unicode-placeholder`?

Seems like `--unicode-placeholder` method prints characters. So I assume once images are printed with `--unicode-placeholder`, they cannot be cleared?

**Comment by kovidgoyal** [comment, marked as answer]
Run the clear command. Basically these are text so dlete them as you
would any other text.
